crates/starknet-devnet-server/src/api/http/models.rs

```diff
@@ -86,8 +86,8 @@ pub struct SerializableAccount {
 
 #[derive(Serialize)]
 pub struct Balance {
-    amount: BigUint,
-    unit: String,
+    pub amount: String,
```

Comment:
why the type is changed to String.

Reply:
From what I saw, this struct was unused so far so it didn't matter if I change it, and I was sure I saw that in some other response we have amount as `String`, but I can't seem to find it anymore. So I guess I should change it back 
 to `BigUint`

Reply:
Reverted in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/394/commits/f80d22eaa65353c532491d2bd1c2be1f4c0866a5